### DISCUSSION TITLE
PEM upgrades

### DISCUSSION THREAD

**Discussion by eckelsjd** [JANUS-Institute/HallThrusterPEM#7] [Ideas]
# General idea
There is no "top-level" model that perfectly fits Hall thruster data. It isn't even guaranteed that a fully kinetic, 3d simulation of the entire thruster would fit experiments, not to mention that would be an unbelievably huge computational problem. You could list hundreds of things that are required for a "true physics" model, but no one knows any for sure. So we take a "bottom-up" approach instead, and iterate on cheaper models to gradually refine the model where its needed. We measure our progress by continually improving on a set of benchmarks.

With that said, it is good to keep in mind the most common modeling topics in the community to guide our iterations, as well as our own results and intuition (i.e. _where_ is model refinement needed and _where_ should we target our efforts). With the bottom-up approach, we can rapidly provide evidence either for or against certain modeling choices, based on UQ and sensitivity analysis. 

To this end, we can use this discussion area to discuss future model iterations for the PEM. Below are listed some ideas to get started.

## Topics from the literature

Here are a few topics from the literature that could use some analysis/justification:

- **Electrostatic assumption** - Nearly all Hall thruster codes use a hard-coded, static magnetic field. Is this justified given all the oscillating plasma currents in the discharge? What about magnetic properties of the wall material and propellant?
- **Near-wall conductivity (NWC)** - To what extent can these effects be ignored? This includes secondary electron emission from ion bombardment of the channel walls and even electron emission from high energy photons from the plasma. Some studies have declared these insignificant, but wall losses and plasma sheath effects are important?
- **Dimensionality** - What radial effects justify a 2d model? What azimuthal effects justify even a 3d model? For example, how different are wall losses when modeled with a 1d code vs. a 2d code?
- **Kinetic effects** - To what extent are fluid (continuum) approximations valid? It's been widely shown that anomalous electron transport is likely due to kinetically-induced turbulence. In which case, what are the best-justified closure models or are there better ways to include the kinetic effects?
- **Chemical reactions** - What ionization/chemical processes are important to capture? Even a fully kinetic Boltzmann equation requires some model for the collision integral and cross-species interactions.
- **Facility effects** - How sensitive is the model to things like background pressure? How sensitive does it need to be to match experimental data? How can other facility effects be incorporated (sputtering, electrical coupling, etc.) and what experimental data is even available to justify these additions?

## Topics from our experience

Here are a few topics from our current experience that motivate further analysis:

- **Anomalous transport** - This closely relates to kinetic effects above. It seems the model is completely dominated by choices for anomalous transport models, and it seems the fluid approximation is a very strong assumption on Hall thruster plasma. Can we find a general-purpose fluid closure that works at least over some small range of operating conditions?
- **Pressure effects** - We cannot capture most experimental trends with background pressure. So thermal ingestion, shifted acceleration zone, etc. models need to be revised.
- **Time-resolution** - It seems likely that time-averaged analysis is too coarse for predicting some highly oscillatory quantities, like discharge current. Frequencies and stability metrics are typically also of importance other than just time-averaged magnitude.

**Comment by archermarx** [comment]
A few others (will add to as I think of them)
- **Improved wall losses:** the wall loss model of HallThruster.jl can be improved. In particular, a more principled treatment of the edge-to-center density ratio would be worth investigating (see https://iopscience.iop.org/article/10.1088/0963-0252/24/2/025017/pdf), as well as improved ion wall losses. This could help reduce the thrust to be more in line with experiment. Lastly, we may be underestimating the power loss to the anode. Fixing this might further reduce thrust.
